### ISSUE TITLE
How to use the configuration file

### ISSUE THREAD

**Issue by batisteo** [Aloxaf/silicon#155]
```bash
$ silicon --config-file
error: Found argument '--config-file' which wasn't expected, or isn't valid in this context

```

**Comment by Aloxaf** [comment]
Oh sorry, this feature only available in 0.4.1-beta.

**Comment by almas1992** [comment]
> Oh sorry, this feature only available in 0.4.1-beta.

Do command line parameters and configuration files have priority?

For example, I set font A in the config file, and set font B via command argument
